graphene_django/fields.py

```diff
@@ -90,6 +90,10 @@ def connection_resolver(cls, resolver, connection, default_manager, max_limit,
         if isinstance(iterable, QuerySet):
             if iterable is not default_manager:
                 default_queryset = maybe_queryset(default_manager)
+                if default_queryset.query.distinct and not iterable.query.distinct:
+                    iterable = iterable.distinct()
```

Comment:
How does this behave when the _other_ distinct call had field args? (`queryset.distinct('field_1', 'field_2')`)

Reply:
I honestly have no idea. From my superficial understanding it should be fine :). Do I understand it correctly that this is only possible when using postgres backends?